python/kwiver/arrows/core/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+set( core_python_headers
+     render_mesh_depth_map.h
+     transfer_bbox_with_depth_map.h
+   )
+
+ set( core_python_sources
+      render_mesh_depth_map.cxx
+      transfer_bbox_with_depth_map.cxx
+      core_module.cxx
+    )
+
+kwiver_add_python_library( core
+   arrows/core
+   ${core_python_headers}
+   ${core_python_sources}
+)
+
+target_link_libraries( python-arrows.core-core
+  PUBLIC ${PYTHON_LIBRARIES}
+         vital
+         kwiver_algo_core
+)
```

Comment:
From a purely C++ perspective, it looks like vital and kwiver_algo_core can be PRIVATE links since there are no header references to those packages. I'm not sure if that changes when doing the python bindings though.

Reply:
Pybind11 is a header only library.